app/views/videos/show.html.haml

```diff
@@ -6,8 +6,38 @@
       .video_info.col-sm-3
         %header
           %h3= @video.title
-          %span Rating: 4.5/5.0
+          %span= "Rating: #{@video.average_rating_of_reviews}/5.0"
```

Comment:
If there are no reviews, it will just show `/5.0`. Is that what you want? Another way would be to show `n/a` or something similar. Either way is fine, I just thought I would give you another idea.